src/mscorlib/src/System/PinnableBufferCacheEventSource.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    internal sealed class PinnableBufferCacheEventSource
```

Comment:
This class doesn't really seem to do anything (also doesn't inherit from EventSource), is this intentional?

Reply:
Wow- good catch. I presume this was an oversight. I'll add it in.